package.json

```diff
@@ -41,10 +44,11 @@
     "documentation": "^5.3.3",
     "es6-promisify": "^5.0.0",
     "eslint": "^4.13.0",
-    "eslint-config-aegir": "^1.0.1",
+    "eslint-config-aegir": "git+https://github.com/ipfs/eslint-config-aegir#flow",
```

Comment:
This needs to link to a released package instead of GitHub URL

Reply:
For technical or practical/policy reasons? I'd prefer not publishing temporary forks.

Reply:
For reproducibility and not breaking stuff in the future. If Github goes down (happens strangely often) or the branch changes, it'll break the assumption that if package.json hasn't changed, we should get the same (~) dependencies. This has bitten us in the past, when cleaning up old repositories but forgetting to update some reference and cases where we want to run tests on old commits.

You can use Prelease tags in npm to publish draft-versions of packages: https://docs.npmjs.com/misc/semver#prerelease-tags

Reply:
So I could do a `1.0.1-flow.1` release and it won't do any harm for anyone not specifying exactly this version?